README.md

```diff
@@ -86,13 +86,14 @@ need them.
 While Rustls itself is platform independent, it uses
 [`ring`](https://crates.io/crates/ring) for implementing the cryptography in
 TLS. As a result, rustls only runs on platforms
-supported by `ring`. At the time of writing, this means x86, x86-64, aarch64,
-armv7, powerpc64le, riscv64gc and s390x. For more information, see [the
-supported `ring` CI targets][ring-ci-targets].
+supported by `ring`. At the time of writing, this means 32-bit ARM, Aarch64 (64-bit ARM),
+x86, x86-64, LOONGARCH64, 32-bit & 64-bit Little Endian MIPS, 32-bit PowerPC (Big Endian),
```

Comment:
I think they want people to write it as "LoongArch" or "LoongArch64"

Reply:
:+1: Fixed. Thanks.

Reply:
Fixed, thanks.